rbac/common/sawtooth/rbac_payload.py

```diff
@@ -61,9 +61,9 @@ def unmake_payload(payload):
         message = protobuf.role_transaction_pb2.CreateRole()
     elif message_type == RBACPayload.PROPOSE_ADD_ROLE_MEMBERS:
```

Comment:
Found an instance of members left unchanged (should be member).

Reply:
thanks, fixed